.runpod/hub.json

```diff
@@ -935,7 +935,7 @@
           "name": "Max Concurrency",
           "type": "number",
           "description": "Max concurrent requests per worker. vLLM has an internal queue, so you don't have to worry about limiting by VRAM, this is for improving scaling/load balancing efficiency",
-          "default": 300,
+          "default": 30,
```

Comment:
This is too low. is there any reason we want to decrease it. What does it solve? May be let's keep it to 50 or 100.